preparedata/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/preparedata/common/InputDataFormatting.groovy

```diff
@@ -145,59 +154,56 @@ IProcess inputDataFormatting(){
             def veget = 'VEGET'
             def vegetStatsZone = 'VEGET_STATS_ZONE'
             def vegetStatsExtZone = 'VEGET_STATS_EXT_ZONE'
+            def impervious = 'IMPERVIOUS'
 
             //Run the sql script
             datasource.executeScript(getClass().getResourceAsStream('inputDataFormatting.sql'),
```

Comment:
@gpetit Same thing for the handling of the method output.

Reply:
ok. done